modules/ctrl_dbus/module.mk

```diff
@@ -5,15 +5,19 @@
 #
 
 MOD		:= ctrl_dbus
-$(MOD)_SRCS	+= ctrl_dbus.c baresipbus.c
+$(MOD)_SRCS	+= baresipbus.c ctrl_dbus.c
```

Comment:
this change should not be needed.

the order of .c files in SRCS should not matter

Reply:
- Tried on my dev host and found you must be right.
- Changed this in the commit.
- But travis tells:
```
modules/ctrl_dbus/ctrl_dbus.c:65:10: fatal error: baresipbus.h: No such file or directory
 #include "baresipbus.h"
          ^~~~~~~~~~~~~~
```
It is plausible, since the Makefile target baresipbus.h is never referenced.

With the reorder baresipbus.h is generated with baresipbus.c. But it is not a beautiful solution. How should I reference the header file correctly?